src/install/aim/intro.mdx

```diff
@@ -0,0 +1,20 @@
+---
+headingText: Introduction to AIM install process
+componentType: default
+---
+
+import apmAimTraceWaterfallView from 'images/apm_screenshot-crop_aim-trace-waterfall-view.webp'
+
+AI Monitoring (AIM) lets you track the performance of the different apps, services, and databases that make up your AI assistant. When your AI responds to a prompt, the response contains a payload, which is made up of bits of data and decision points that New Relic can monitor. Instrumenting your AI-powered app allows us to ingest that data, letting you monitor and improve your AI assistant.
```

Comment:
>  When your AI responds to a prompt, the response contains a payload, which is made up of bits of data and decision points that New Relic can monitor

We might want to reword this section a little bit. We don't monitor the customer's AI's response payload to determine what happened. Maybe something like this (wordsmithed to actually sound good)

> When your AI receives a prompt, our New Relic agents can trace the process it used to come up with an answer, through external LLMs, vector stores, and more